### PR TITLE
Adds enhanced support for proxy models.

### PR DESCRIPTION
**Pull request by abettke**
- Allows for proxy models to be used directly in DjangoObjectType definitions.
- Updates tests for proxy model support.
- Removes test showing proxied DjangoObjectType definitions fail.

### CODE REVIEW

File: graphene_django/tests/test_query.py [hunk 1]
```diff
@@ -996,14 +947,23 @@ class Query(graphene.ObjectType):
         }
     """
 
+    def str_to_node_id(val):
+        return base64.b64encode(val.encode()).decode()
```
Comment (zbyte64) [inline graphene_django/tests/test_query.py]: I think we should use `graphql_relay.to_global_id` here.

Comment (abettke) [reply graphene_django/tests/test_query.py]: :thumbsup: done

**Comment by abettke** [comment]
> Does this need any documentation to go along with it?

None that I can think of. This just adds, what seems to me, expected behavior. Beforehand, using a proxy model in a DjangoObjectType threw a GraphQLError. Now, it fully supports using proxy models the same way that base models are used.

**Comment by threewordphrase** [comment]
I'm waiting with bated breath for this :)

**Comment by abettke** [comment]
For anyone needing this immediately, my fork contains theses changes, and as far as I know is currently up to date with master.

**Comment by IAmNatch** [comment]
Note to anyone searching: Upgrading from 2.2.0 to 2.4.0 resolved my issue with serializing a taggit proxy model. Thanks all who worked on this!

**Pull request merged**